locals.tf

```diff
@@ -28,6 +28,10 @@ locals {
       override                   = true
       preload                    = true
     }, var.cloudfront.hsts)
+    response_headers_to_remove = merge({
+      server   = false,
+      opennext = true
+    }, var.cloudfront.response_headers_to_remove)
```

Comment:
I'd rename this to `remove_headers_config` and have it as:
```tf
remove_headers_config = merge({
    items: []
}, var.cloudfront.response_headers_config)
```

Reply:
done